modules/lambda-registrator/main.tf

```
@@ -126,9 +154,34 @@ resource "aws_iam_role_policy_attachment" "lambda_logs" {
   role       = aws_iam_role.registration.name
   policy_arn = aws_iam_policy.policy.arn
 }
+resource "random_id" "repo_id" {
+  byte_length = 8
+}
+resource "aws_ecr_repository" "lambda-registrator" {
+  count        = var.ecr_image_uri != "" ? 0 : 1
+  name         = local.ecr_repo_name
+  force_delete = true
+}
+
+
+resource "docker_image" "lambda_registrator" {
+  count = var.ecr_image_uri != "" ? 0 : 1
+  name  = var.consul_lambda_registrator_image
+}
```

general style nit: we should keep a consistent spacing of 1 newline between blocks within the TF files.

```suggestion
resource "random_id" "repo_id" {
  byte_length = 8
}

resource "aws_ecr_repository" "lambda-registrator" {
  count        = var.ecr_image_uri != "" ? 0 : 1
  name         = local.ecr_repo_name
  force_delete = true
}

resource "docker_image" "lambda_registrator" {
  count = var.ecr_image_uri != "" ? 0 : 1
  name  = var.consul_lambda_registrator_image
}
```

---

For the `byte_length` of the `random_id`, what do you think about using 4 bytes instead of 8? I think the risk of the collision is super low and the name prefix `consul-lambda-registrator-` is already pretty long.